modules/editor/editor.js

```diff
@@ -1,8 +1,45 @@
+import InserterButton from './inserter/button';
+
+const h = wp.element.createElement;
+
+const EditorComponent = ( { content, state: { inserter }, toggleInserter } ) => {
```

Comment:
Also destructuring any more than one level deep quickly becomes unreadable. I think this one isn't so bad but we should be cautious of it.